tests/functional/end-to-end.bats

```diff
@@ -267,7 +271,7 @@ test_one_check() {
 
 	run ${SELINT_PATH} -c configs/default.conf policies/misc/disable_require_start.*
 	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
+	[ "$output" == "disable_require_start.te:8: (W): Gen require block unquoted (W-010)" ]
```

Comment:
Can you just add a "-d W-010" to the selint call instead?

Reply:
yup